### PR TITLE
Feature/gat 341

### PR DESCRIPTION
**Pull request by ThinkingMonkeyLtd**

### CODE REVIEW

File: src/hooks/usePersistState/useDomChanged.stories.mdx [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import { Meta, Story, Canvas } from '@storybook/addon-docs';
```
Comment (ThinkingMonkeyLtd) [inline src/hooks/usePersistState/useDomChanged.stories.mdx:1]: Change file name

File: src/pages/collections/CollectionsSearch.js [hunk 2]
```diff
@@ -1,12 +1,10 @@
 import React, { useState } from 'react';
```
Comment (ThinkingMonkeyLtd) [inline src/pages/collections/CollectionsSearch.js:1]: Not used?

File: src/pages/dashboard/Components/AccountDatasets/AccountDatasetsContent.js [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+			console.log('asdasdasd', team, dataset.listOfVersions);
+			return dataset.listOfVersions.length > 0 && team === 'admin';
+		},
+		[team]
+	);
+
+	const getDatasetCardProps = dataset => {
+		const datasetCardProps = {};
+
+		console.log('dataset', dataset.activeflag === 'inReview', hasActivityHistory(dataset));
```
Comment (ThinkingMonkeyLtd) [inline src/pages/dashboard/Components/AccountDatasets/AccountDatasetsContent.js:39]: Remove

File: src/pages/dashboard/Components/AccountDatasets/AccountDatasetsCreate.js [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+			<div className='accountHeader'>
+				<Row>
+					<Col sm={12} md={8}>
+						<div>
+							<span className='black-20'>Datasets</span>
+						</div>
+						<div>
+							<span className='gray700-13 '>
+								{publisherID !== 'admin'
+									? 'View, add, edit, archive and check the status of your datasets.'
```
Comment (ThinkingMonkeyLtd) [inline src/pages/dashboard/Components/AccountDatasets/AccountDatasetsCreate.js:55]: Change translations

File: src/pages/dashboard/Components/AccountDatasets/AccountDatasetsCreate.js [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+					<Col sm={12} md={4} style={{ textAlign: 'right' }}>
+						{team !== 'admin' && (
+							<Button
+								variant='primary'
+								className='addButton'
+								onClick={
+									(() => googleAnalytics.recordEvent('Datasets', 'Add a new dataset', 'Datasets dashboard button clicked'),
+									createNewDataset)
+								}>
+								+ Add a new dataset
```
Comment (ThinkingMonkeyLtd) [inline src/pages/dashboard/Components/AccountDatasets/AccountDatasetsCreate.js:69]: Change translations

**Review by kandaj — APPROVED** [review]

**Pull request merged**